src/main/java/dansplugins/recipesystem/MoreRecipes.java

```diff
@@ -8,16 +8,20 @@
 
 public final class MoreRecipes extends JavaPlugin {
 
+    private static MoreRecipes instance;
+
     // version
-    public static String version = "v1.4";
+    public String version = "v1.4";
```

Comment:
Shouldn't this be a `final` ?

IMHO it shouldn't have the `v` either.

Reply:
Thanks for your review! This can definitely be final, I'll go ahead and make that change. As far as the "v" goes, that is consistent with the rest of my plugins.